mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -27,14 +27,15 @@
     WdcParsimoniousDimensionParser,
 )
 from mowgli_etl.pipeline.wdc.wdc_offers_corpus_entry import WdcOffersCorpusEntry
+from mowgli_etl.pipeline.wdc.wdc_offers_corpus import WdcOffersCorpus
 
 
 class WdcTransformer(_Transformer):
 
     def transform(
         self,
         *,
-        wdc_jsonl_file_path: Path,
+        wdc_corpus: WdcOffersCorpus,
```

Comment:
I like this.

As an aside, you don't need to use the wdc_ prefix for "internal" things. It's just for file names, class names, and anything else exported from the top level of a module (like WDC_* constants), which are ~ a global namespace. So this could be corpus, product_type_classifier, and so on.